rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.3.4-release-notes.adoc

```diff
@@ -0,0 +1,47 @@
+= RPA Bot 1.3.4 Release Notes
+
+RPA Bot is a software robot that runs in an on-premise environment and executes the assigned automations.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== October 3, 2023
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* RPA Bot now enables DEBUG level logging for StSec, StSem, and FreeRDP when it gets an EnableDebugLogLevelTask.
+* Bot UUID is added to Bot logging System infos.
+* RPA Bot now logs scheduler information when it goes into suspended status from the scheduler.
+* StSec RunID creation is moved from StSec to RPA Bot so that RPA Bot now creates and logs StSec RunID and passes it to the StSec.
+* When the logging level warning is enabled, FreeRDP is logged by default.
```

Comment:
That's not correct. FreeRdp logging is now enabled by default.
So, a description like this would be more precise:
"FreeRDP logging level WARN is now enabled by default."